docs/docs/creating-global-styles.md

```diff
@@ -2,7 +2,124 @@
 title: Creating global styles
 ---
 
-In this guide, we'll walk through using traditional global styles in your site.
+In nearly every site, there will be some global styles, such as a reset or typography defaults. This guide will walk through how to add global styles to your site, whether you use standard `.css` files or a CSS-in-JS solution.
+
+## How to add global styles in Gatsby with standard CSS files
+
+In general, the best way to add global styles is with a [shared layout component](https://next.gatsbyjs.org/tutorial/part-three/#your-first-layout-component). This layout will be used in every page component to apply global styles (and often shared components such as a header and/or footer).
```

Comment:
Doesn't need "In general".

Reply:
*This layout component

Reply:
The second sentence is a bit awkward. I'd frame the layout component as a component for things that's shared across different pages so stuff like styles & headers, etc.